Source/Core/Math/ICalculator.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Core.Math
+{
+    public interface ICalculator
```

Comment:
As already discussed, it is not necessary to implement it as an interface. Consider to implement a static class.